src/renderer/containers/ReplayFileStats/index.tsx

```diff
@@ -11,20 +13,15 @@ export interface ReplayFileStatsProps {
   onClose: () => void;
 }
 
-export const ReplayFileStats: React.FC<ReplayFileStatsProps> = ({
-  file,
-  index,
-  total,
-  onNext,
-  onPrev,
-  onClose,
-}) => {
+export const ReplayFileStats: React.FC<ReplayFileStatsProps> = (props) => {
+  // const { file, index, total, onNext, onPrev, onClose } = props;
+
   const loading = useReplays((store) => store.selectedFile.loading);
   const error = useReplays((store) => store.selectedFile.error);
   const gameStats = useReplays((store) => store.selectedFile.gameStats);
   return (
     <div>
-      <div>{file.name}</div>
+      {/* <div>{file.name}</div>
```

Comment:
The `onClose` event needs to be handled rather than just commented out. Since we no longer have a "back" button, we need a way to go back from the stats page to the replay file list.